fv_be/app/admin/characters_admin.py

```diff
@@ -13,11 +13,13 @@ class CharacterAdmin(BaseSiteContentAdmin):
         "approximate_form",
         "site",
         "notes",
+        "related_dictionary_entries",
     )
     list_display = (
         "title",
         "sort_order",
         "approximate_form",
+        "related_dictionary_entries",
```

Comment:
I believe for a ManyToMany field the list_display will give an error. There are workarounds for adding a ManyToMany field to the list view but it may be better to leave that field as only editable from the detail view.